fix: font size during landscape orientation

andymcheng opened the pull request.

berkalpyakici committed:
fix: add code style

andymcheng committed:
fix for font size being too large in landscape mode on iphone safari browser

andymcheng committed:
[wip] fix: landscape mobile browser font size

TomWoodward commented on generic-styles/index.less:
```diff
@@ -28,6 +28,11 @@
       margin-inline-start: 0;
     }
   }
+  code {
```
need to remove this from this PR, its on the other one

TomWoodward commented on generic-styles/index.less:
```diff
@@ -41,3 +46,6 @@
   }
 }
+html {
+  -webkit-text-size-adjust: 100%; /* Prevent font scaling in landscape while allowing user zoom */
```
this is fine, but can we investigate maybe using the viewport meta tags instead?

TomWoodward replied:
you can use `text-size-adjust` without the webkit prefix, it'll figure it out

TomWoodward committed:
Merge branch 'master' into fix-iphone-font-orientation

TomWoodward committed:
Merge branch 'master' into fix-iphone-font-orientation

andymcheng committed:
removed unncessary webkit

andymcheng committed:
Merge branch 'fix-iphone-font-orientation' of https://github.com/openstax/rex-web into fix-iphone-font-orientation

TomWoodward committed:
Merge branch 'master' into fix-iphone-font-orientation

TomWoodward reviewed (changes requested):
i totally missed this on my first review, but since this fix is for the whole app and on the `html` and not generic styles specific, please add it to `src/index.css` instead of here

TomWoodward committed:
Merge branch 'master' into fix-iphone-font-orientation

andymcheng committed:
migrated change to src/index.css

moved text-size-adjust fix from index.less file to src/index.css file as it applys to the whole app

TomWoodward committed:
Merge branch 'master' into fix-iphone-font-orientation
